[EDU-3924] feat: metrics requests edge app (#672)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx

You can request to increase the limits based on your plan. Contact the [technical support team](/en/documentation/services/support/) to request it.
:::

---

## Billing

**Edge Application** has specific metrics that define how the product is billed.

<Button href="https://www.azion.com/en/pricing/#EdgeApplication" text="see pricing"></Button>

### Requests

> Total amount of HTTP and HTTPS requests from an user or device made to edge applications running on Azion Edge Computing Platform. A request is billed independently of request success or failure and version of server protocol (HTTP/1.1, HTTP/2.0, HTTP/3.0).

Reviewer: ```suggestion
Total amount of HTTP and HTTPS requests from an user or device made to edge applications running on Azion Edge Computing Platform. A request is billed independently of request success or failure and server protocol version (HTTP/1.1, HTTP/2.0, HTTP/3.0).
```

> When a content, such as an image cached at the edge is requested, it isn't billed.

Reviewer 2: ```suggestion
When content, such as an image cached at the edge, is requested, it isn't billed.
```

> When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the amount of filters being used. For example, if the request queries three filters, it'll be billed as one request.

Reviewer: ```suggestion
When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the amount of query strings being used to apply filters. For example, if the request queries three filters, it'll be billed as one request.
```

Each filter is a query string (`?filter=value&filter2=value2`), maybe this is clearer.

> When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the amount of filters being used. For example, if the request queries three filters, it'll be billed as one request.

Reviewer 2: ```suggestion
When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the number of filters being used. For example, if the request queries three filters, it's billed as one request.
```
